### PR TITLE
Feature: configurable footer on welcome page just as cool as the one on meet.jit.si

### PR DESCRIPTION
**Pull request by florisvangeel**
## Problem statement:
In recent times it's become more important to have good privacy statement and terms of conditions.  
When self hosting the regular method of facilitating this by adding these links to static/welcomePageAdditionalContent.html
Then manipulating the DOM with css. this is not a clean way of doing.  

## Solution
Adopted react/features/welcome/components/WelcomePage.web.js to allow for configuration flags from interface_config.js. 
Now we can all use this beautiful footer and customise it to our (clients) needs.  
Notice that instagram might not be the official social media and social links that are not available can be commented.  

## Steps to test this code
clone the pull request use node 14.x and npm@7
```
npm install
export WEBPACK_DEV_SERVER_PROXY_TARGET=https://test.plusteam.live/
make dev
```
Results look like this:
image

## Footnote
Thank you all for building and maintaining this awesome free and open source software.

### CODE REVIEW

File: react/features/welcome/components/WelcomePage.web.js [hunk 1]
```diff
@@ -201,57 +201,6 @@ class WelcomePage extends AbstractWelcomePage {
...
-                        <h1 className = 'header-text-title'>
-                            { t('welcomepage.headerTitle') }
-                        </h1>
-                        <span className = 'header-text-subtitle'>
-                            { t('welcomepage.headerSubtitle')}
-                        </span>
-                        <div id = 'enter_room'>
-                            <div className = 'enter-room-input-container'>
-                                <form onSubmit = { this._onFormSubmit }>
-                                    <input
```
Comment (saghul) [inline react/features/welcome/components/WelcomePage.web.js:213]: Where is this now?

Comment (idevit) [reply react/features/welcome/components/WelcomePage.web.js]: Sorry i lost that part due to local customisation.  
reverted in next commit.

**Comment by idevit** [comment]
Re submitted CLA docusign (not sure if the previous from jitsi hackathon is still active)

**Comment by idevit** [comment]
in my opinion this feature is complete.  
please let me know if something else is needed?

**Pull request closed without merging**